ui/src/app/pages/login/account-creation/common.tsx

```diff
@@ -158,3 +160,21 @@ export const OptionalDropDownSection = (props) => {
   </DropDownSection>;
 };
 
+export const getRoleOptions = (institutions: Array<PublicInstitutionDetails>, institutionShortName: string):
```

Comment:
Hooray for reuse.  Should this go up a level or two?  A little weird for Profile Edit to be referring to this file.  Maybe some kind of institution/common